os/kernel/init/os_start.c

```diff
@@ -825,6 +825,11 @@ void os_start(void)
 
 	DEBUGVERIFY(os_smp_start());
 
+	dbg("All SMP cores started!!\n);
+#ifdef CONFIG_AMP
+	dbg("AMP configuration is enabled. Affinity of all tasks set to core 0!!!\n");
```

Comment:
How about adding "by default" at the end? I think it will be more clear. If someone sets the affinity to cpu1 in some task and then sees this message, it will cause some confusion.
```suggestion
	dbg("AMP configuration is enabled. Default affinity of task is set to CPU0!!!\n");
```